composite-actions/nodejs-script/service-definition-validator/src/types/index.ts

```diff
@@ -0,0 +1,42 @@
+import type { YAMLMap } from "yaml";
+
+export type Environment = "staging" | "production";
+export type ServiceType = "INPUT_API" | "STATEFUL_PROCESSOR" | "STATELESS_PROCESSOR" | "QUERY_API";
```

Comment:
I am not sure how this works in node.js but these are defined elsewhere already. Should they be referenced instead?